auth/migrations/0004_user_ids_part2.sql

```diff
@@ -0,0 +1,27 @@
+-- All rows should have user_ids at this point
```

Comment:
Nit: should we have a comment about how this was done in the application? So someone who reads the migrations down the line won't be confused.